fml/concurrent_message_loop.h

```diff
@@ -72,7 +73,9 @@ class ConcurrentTaskRunner : public BasicTaskRunner {
  private:
   friend ConcurrentMessageLoop;
 
-  std::weak_ptr<ConcurrentMessageLoop> weak_loop_;
+  // Raw pointer that is cleared out in ~ConcurrentMessageLoop.
+  ConcurrentMessageLoop* weak_loop_;
```

Comment:
Consider adding a macro like `IPLR_GUARDED_BY` to `fml/macros.h` and annotating this.

Reply:
(and defining the mutex first)

Reply:
`IPLR_GUARDED_BY` is impeller only, its in `//impeller/base/thread_safety.h`

Reply:
Yes, which is why I'm saying to add something like `FML_GUARDED_BY` to `//fml/macros.h`

Reply:
Done

Reply:
Sorry, now it's done (i had added it but forgot to commit the change where it is used).